docs/guides/create-basic-component.stories.mdx

```diff
@@ -85,6 +93,8 @@ Right, I promised to explain them.
 | `AmountCounter.test.tsx`    | Self-explanatory.                                                                                                                                                                                           |
 | `AmountCounter.css`         | Self-explanatory.                                                                                                                                                                                           |
 | `AmountCounter.css.d.ts`    | This wasn't on the original list but it's very important. This is how react knows what class names you added and automagically serializes them so it doesn't clash with other components. More on it later. |
+| `AmountCounter.stories.mdx` | This file is the front page of the component documentation where you will describe the main features of it.                                                                                                 |
```

Comment:
<img width="784" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/d15ab94f-2be1-46f8-b347-ad8cbf5035ec">
This was already present on line 84

Reply:
Oohhh, I didn't even notice. Thanks for catching it. I moved the `Web.stories.tsx` to the line below the `AmountCounter.stories.mdx`